README.md

```diff
@@ -16,16 +16,18 @@ Check out the [Terraform](https://developer.hashicorp.com/terraform/language/bac
 locals {
   region = "eu-west-1"
   spacelift_version = "v3.4.0"
-  website_domain    = "https://spacelift.mycorp.io"
-  mqtt_domain       = "tls://spacelift-mqtt.mycorp.io:1984" # Must start with `tls://` and end with a port number
+  website_domain    = "spacelift.mycorp.io"
+  website_endpoint  = "https://${local.website_domain}"
+  mqtt_domain       = "spacelift-mqtt.mycorp.io"
+  mqtt_endpoint     = "tls://${local.mqtt_domain}:1984"
 }
 
 module "spacelift_infra" {
   source = "github.com/spacelift-io/terraform-aws-spacelift-selfhosted?ref=v1.0.0"
 
   region         = local.region
   default_tags   = {"app" = "spacelift-selfhosted-infra", "env" = "dev"}
-  website_domain = local.website_domain
+  website_endpoint = local.website_endpoint
```

Comment:
the spacing needs to be a bit readjusted here, right?

Reply:
Fixed.